app/components/dashboard/images/row.svg

```diff
@@ -0,0 +1,7 @@
+<svg width="23px" height="18px" viewBox="0 0 23 30" version="1.1" xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink">
```

Comment:
Should viewBox also be "0 0 24 24"? I know there is something to do with standard sizing but it may not relate to this svg.

Reply:
If I put 0 0 24 24 then it crops the bottom of the svg, so I figured that the width and height needs to be different for this one. After playing with different sizes I found that this combination worked the best and looks the same size as the other icons.